### DISCUSSION TITLE
Where can I find Stellariuim logs after crashing?

### DISCUSSION THREAD

**Discussion by jairoamaral** [Stellarium/stellarium#3523] [Q&A]
Stellarium is crashing while I control my telescope mount with it. I would like to read the logs to find out what is happening, but I don't know were they are stored, or even if they are being written.

Please, what is the path to find them them?

**Comment by alex-w** [comment]
Stellarium User Guide, chapter 5 (esp. paragraphs 5.1 and 5.3)

**Reply by jairoamaral** [reply]
I found ony one log.txt, dated from today.

Is all the useful information erased every time I start Stellarium again, so I need to pick this log right afther a crash?

**Reply by gzotti** [reply]
yes

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/FAQ#i-dont-read-documentation-where-is-that-configini-file-or-logfile-you-are-always-pointing-to

**Comment by gzotti** [comment]
Also read the Release Notes about crashes with the TC plugin and what you can do about them!

**Reply by jairoamaral** [reply]
I could only find this:
"Fixed crash in Telescope Control plugin for Qt6-based packages (GH: #2874, #2821)" and it's from 2022. Unsolved yet?

I'm using a ZWO AM5 mount. Does it relate?

**Reply by gzotti** [reply]
From http://stellarium.org/release/2023/09/25/stellarium-23.3.html:
> TELESCOPE USERS

>We have identified and fixed some bugs in the TelescopeControl plugin on the Qt6-based releases.

>Telescope users, please try whether this solves your problems. If not, we are looking for those of you with programming skills to help us fixing remaining issues. Reward possible!

We do not have your mounts and don't know what protocols they are using and why that fails sometimes. If you are affected, use a Qt5-based build of 23.3.
